source/slang/ir-insts.h

```diff
@@ -33,6 +33,11 @@ struct IRLayoutDecoration : IRDecoration
     enum { kDecorationOp = kIRDecorationOp_Layout };
 
     RefPtr<Layout>  layout;
+    virtual void dispose() override
+    {
```

Comment:
I'm not clear on why the `IRDecoration` hierarchy is using a manual `dispose` operation instead of just letting a auto-generated destructor clean up everything. What am I missing?